### ISSUE TITLE
cupsd Segmentation fault on CloseUI without matching OpenUI

### ISSUE THREAD

**Issue by severach** [OpenPrinting/cups#64]
Brother MFC-J6710DW and MFC-J6910DW ppd have a section like this.

````
*%=== Reverse Printing ================================
*%OpenUI *BRReverse/Reverse Printing: PickOne
*%OrderDependency: 30 AnySetup *BRReverse
*%DefaultBRReverse: OFF
*%BRReverse OFF/Off: "          "
*%BRReverse ON/On: "          "
*CloseUI: *BRReverse
````

`OpenUI` is commented out. `CloseUI` is not commented out. In the past this was silently ignored. Now it crashes `cupsd`. For now the solution is to patch a comment `%` into the unwanted `CloseUI`.

````
# pacman -Q cups
cups 1:2.3.3op1-1
````
[brother_mfcj6710dw_printer_en.ppd.segfault.zip](https://github.com/OpenPrinting/cups/files/5677152/brother_mfcj6710dw_printer_en.ppd.segfault.zip)
[brother_mfcj6910dw_printer_en.ppd.segfault.zip](https://github.com/OpenPrinting/cups/files/5677153/brother_mfcj6910dw_printer_en.ppd.segfault.zip)

**Comment by michaelrsweet** [comment]
Ugh, this won't happen normally but we definitely want to fix it!

**Comment by pranshukharkwal** [comment]
@michaelrsweet I tried looking for the exact place where the code is breaking. 
I figured that in 'cups/ppd.c' , in line number 1499, `option->defchoice` is causing the issue. 
Apparently this variable is not accessible (probably does not exists) 

How can this issue be fixed?

**Comment by tillkamppeter** [comment]
Fixed via Pull Request #78.
